orttraining/orttraining/python/training/amp/loss_scaler.py

```diff
@@ -0,0 +1,88 @@
+class LossScaler(object):
+    r"""Base class for implementing custom loss scaler strategies
+
+    Once the scaler is configured, no user intervention is needed to update loss scale during training.
+
+    Note:
+        This class should never be instantiated, but used as an abstract class for custom implementation
```

Comment:
Can we say "...custom loss scaling strategy."

Reply:
Sure, sometimes line length gets in the way, but your suggestion is still shorter than 128 chars :)